graph/src/data/query/result.rs

```diff
@@ -174,13 +181,16 @@ pub struct QueryResult {
     data: Option<Data>,
     #[serde(skip_serializing_if = "Vec::is_empty")]
     errors: Vec<QueryError>,
+    #[serde(skip_serializing)]
+    pub deployment: Option<SubgraphDeploymentId>,
```

Comment:
Not very nice that this ended up here, I haven't looked closely but maybe it will be better located in `QueryResults`?

Reply:
I tried that very briefly, but couldn't get it right; I totally agree that it's weird that that is there (it's only needed to report the query execution time for the right deployment in `server/http/service.rs`) I'll try and think of a better way for a followup PR.